inspirehep/modules/workflows/views.py

```diff
@@ -25,14 +25,28 @@
 from __future__ import absolute_import, division, print_function
 
 from os.path import join
-from flask import Blueprint, jsonify, request, current_app
+from functools import wraps
 
+from flask import (
+    Blueprint,
+    current_app,
+    jsonify,
+    request
+)
+from flask.views import MethodView
 from invenio_db import db
+from inspire_schemas.api import validate
 from invenio_workflows import workflow_object_class, ObjectStatus
 from invenio_workflows.errors import WorkflowsMissingObject
+from jsonschema.exceptions import ValidationError
 
 from inspire_utils.urls import ensure_scheme
+from inspirehep.modules.workflows.errors import CallbackMalformedError, \
+    CallbackValidationError, CallbackWorkflowNotFoundError
```

Comment:
We usually use parenthesis to get multiline imports (you did it for flask),

Reply:
I know that's some "styling" stuff, so we can leave it for whenever we decide on that.

Reply:
for the flask it was you :P but I'll update it. 👍 we should decide and automate this kind of decisions

Reply:
I was investigating still :), I'm open to alternatives, I don't have a lot of experience on it.